tests/onnx/sanity_tests/test_classification_sanity_sample.py

```diff
@@ -1,91 +0,0 @@
-"""
```

Comment:
Since this test was dropped, what about the test for the other existing samples for ONNX?

Reply:
I suppose to add the test in ONNX precommit for testing the metrics of the quantized model based on the current mobilenet_v2 sample.
The stopper for now is that we need to download dataset, which has big size for precommit test (4GB).
This test can be added when the dataset size problem is resolved (most probably by changing it)

Reply:
@alexsu52 please correct me If I am wrong